data/datasource.go

```diff
@@ -53,6 +58,7 @@ func init() {
 	addSourceReader("consul+http", readConsul)
 	addSourceReader("consul+https", readConsul)
 	addSourceReader("boltdb", readBoltDB)
+	addSourceReader("aws+ssm+param", readAWSSSMParam)
```

Comment:
This schema feels a bit long, though I'm not sure I can can think much better - perhaps `aws+smp` (`smp` for Systems Manager Parameter - which is what they're called in the docs)?